uwb/example/lib/game_widgets/brick_field.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/material.dart';
+
+import 'brick.dart';
+
+///The [BrickField] class. Generates a field of bricks
```

Comment:
nit: missing a space after the doc-comment specifier and a missing period at the end of the sentence:
```suggestion
/// The [BrickField] generates a field based on a supplied collection of [Brick]s.
```